src/main/java/com/global/inbox/service/impl/ItemServiceStub.java

```diff
@@ -1,15 +1,30 @@
 package com.global.inbox.service.impl;
 
+import com.global.inbox.dto.ItemDto;
 import com.global.inbox.model.Item;
+import com.global.inbox.model.ItemStatus;
+import com.global.inbox.repository.ItemRepository;
 import com.global.inbox.service.ItemService;
+import lombok.AllArgsConstructor;
 import org.springframework.stereotype.Service;
 
+import java.time.LocalDateTime;
 import java.util.List;
 import java.util.UUID;
 
 @Service
+@AllArgsConstructor
 public class ItemServiceStub implements ItemService {
 
+    private ItemRepository itemRepository;
+
+    @Override
+    public Item save(ItemDto itemDto) {
```

Comment:
Mixing scope. From one side, your Stub returns mocked data and at the same moment - works with database. Should be aligned to the ONLY one option